test/bdd/pkg/context/context.go

```diff
@@ -70,7 +75,7 @@ func NewBDDContext(caCertPath, testDataPath string) (*BDDContext, error) {
 	return &instance, nil
 }
 
-func createVDRI(universalResolver string) (vdriapi.Registry, error) {
+func createVDRI(universalResolver, trustBlocResolverURL string) (vdriapi.Registry, error) {
```

Comment:
can you remove universalResolver arg

Reply:
not used in the method

Reply:
It's used by non-trustbloc DID methods https://github.com/trustbloc/edge-service/blob/master/test/bdd/pkg/context/context.go#L74-L78.

Reply:
[Updated](https://github.com/trustbloc/edge-service/compare/a05a8f4c9757ffceeb61e617ab20a4741467469f..0e4f5f2d43393ddb3762060c2763c68efd68e15e)